docs/reference/query-languages/esql/esql-enrich-data.md

```diff
@@ -17,11 +17,12 @@ For example, you can use `ENRICH` to:
 
 [`ENRICH`](/reference/query-languages/esql/esql-commands.md#esql-enrich) is similar to [`LOOKUP join`](/reference/query-languages/esql/esql-commands.md#esql-lookup-join) in the fact that they both help you join data together. You should use `ENRICH` when:
 
-* Enrichment data doesn't changes frequently
+* Enrichment data doesn't change frequently
 * You can accept index-time overhead
 * You are working with structured enrichment patterns
 * You can accept having multiple matches combined into multi-values
 * You can accept being limited to predefined match fields
```

Comment:
The matching logic of "which lookup/enrich document matches a given input row" is also more lenient for ENRICH compared to LOOKUP JOIN. For instance, for `LOOKUP JOIN`, multivalued join keys like `["foo", "bar"]` don't match anything if they occur in the main or lookup index; for `ENRICH`, there'll be a match if any value for the main index' join key is contained in the lookup index' join key.

I'll gather more precise information for this, we can maybe add this in a follow-up PR.